packages/backend/src/studio.ts

```diff
@@ -66,18 +66,34 @@ export class Studio {
     this.#extensionContext = extensionContext;
   }
 
+  private checkVersion(): boolean {
+    if (!version) return false;
+
+    const current = coerce(version);
+    if (!current) return false;
+
+    if (current.major === 0 && current.minor === 0) {
+      console.warn('nighties version are not subject to version verification.');
+      return true;
+    }
+
+    return satisfies(current, engines['podman-desktop']);
+  }
+
   public async activate(): Promise<void> {
     console.log('starting AI Lab extension');
     this.telemetry = env.createTelemetryLogger();
 
-    // Ensure version is above minimum podman desktop version supported
-    if (!version || !satisfies(coerce(version), engines['podman-desktop'])) {
+    if (!this.checkVersion()) {
       const min = minVersion(engines['podman-desktop']);
+      const current = version ?? 'unknown';
```

Comment:
I'm confused why the `??` is necessary since the type of version is `string`, not `string | undefined`?

Reply:
old versions of Podman Desktops we're not providing the version field

Reply:
Ah, that makes sense then. But what about `(version as string) = undefined;` in test? It gives me an error locally since version cannot be undefined in current API.